scripts/talky

```diff
@@ -2,6 +2,17 @@
 
 BASE_URL="https://talky.io"
 
+base64enc() {
+    if [[ -x $(which openssl) ]] ; then
+        retval=$(echo $1 | openssl enc -base64)
+    if [[ -x $(which base64) ]] ; then
+        retval=$(echo $1 | base64)
+    else
+        echo "Unable to base64 encode, install base64?"
```

Comment:
It should probably `exit` here instead, since otherwise this error message will make it through to `matterhorn`.